client/src/pages/Home.js

```diff
@@ -26,16 +26,9 @@ export function Home() {
 	return (
 		<main role="main">
 			<div>
-				<img
-					className="logo"
-					data-qa="logo"
-					src={logo}
-					alt="Just the React logo"
-				/>
-				<h1 className="message" data-qa="message">
-					{message}
-				</h1>
-				<Link to="/about/this/site">About</Link>
+				<div className="container ">
+					<MainTable />
```

Comment:
The two `div`s aren't doing anything—let's remove them, and we can move `className="container"` to the `main` element

Reply:
Ok Coung, I see the problem I am removing it thank you.